Package.swift

```diff
@@ -40,12 +40,12 @@ let package = Package(
   dependencies: [
     // Dependencies declare other packages that this package depends on.
     .package(
-      url: "https://github.com/bourdakos1/abseil-cpp-SwiftPM.git",
-      branch: "cxx17-test"
+      url: "https://github.com/monzo/abseil-cpp-SwiftPM.git",
+      exact: "0.20240902.0"
```

Comment:
This tag doesn't exist yet. Pending on https://github.com/monzo/abseil-cpp-SwiftPM/pull/1